.github/workflows/snyk-scan.yml

```diff
@@ -0,0 +1,115 @@
+name: Snyk Security Vulnerability Scan
+
+on:
+  pull_request:
+  push:
+    tags:
+      - 'RELEASE-[0-9]+.[0-100]+.[0-9]+.[0-9]+-[0-9]+'
```

Comment:
I added this `[0-100]+` since there are double-digit values (eg:34) used currently for tags (eg: RELEASE-3.34.0.4-1 )

This will be triggered for new tags only in the future. Hence it will not trigger (Snyk Monitor) for already available old tags

Reply:
Wouldn't `[0-9]+` do the same job?

Reply:
Hi, it would only match one digit only from 0 to 9. Since the current tags are in the format of "RELEASE-X.**XX**.X.X-X", we had to put [0-100]+ to match "XX" pattern 
(eg: to match 34 in the RELEASE-3.34.0.4-1)

Reply:
Wouldn't the plus sign indicate one or more occurrences of the preceding element (digit)? In other words, `[0-9]+` would match `1` as well as `1000`.

Reply:
Yes got your point. thanks for pointing it out. I will update this

Reply:
Also shouldn't `.` be escaped? In common regexp, '.' matches an arbitrary character.

Reply:
No as per the docs https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags 
There is an example here in the bottom